docs/reference/one-sdk.mdx

```diff
@@ -0,0 +1,134 @@
+import { SuperfaceClient } from '@superface/one-sdk';
+
+const client = new SuperfaceClient();
+```
+
+### Performing a use-case (`getProfile`, `getUsecase`, `perform`)
+
+With client initialized you can perform the use-cases in installed profiles using the following series of methods:
+
+1. `getProfile('<profileName>')` returns an instance of Profile from the SDK client instance,
+2. `getUsecase('<usecaseName>')` returns a use-case from Profile,
+3. `perform(input, [options])` executes the use-case with provided input data and optional `options` argument.
+
+```typescript
+const profile = await client.getProfile('<profileName>');
+const result = await profile.getUsecase('<usecaseName>').perform({
+  inputField: 1,
+  anotherInputField: 'hello',
+});
+```
+
+Perform returns a [result object](#result-object).
+
+### Selecting a provider (`getProvider`) {#get-provider}
+
+By default, OneSDK picks the first configured provider according to the [`priority` array](/reference/superjson#profilesettings) in `super.json` configuration.
+
+You can get a specific provider with `getProvider` method on SDK client and passing it to the perform:
+
+```typescript
+const provider = await client.getProvider('<providerName>');
+const result = await profile.getUseCase('<usecaseName>').perform(
+  {
+    // Input parameters
+  },
+  { provider }
+);
+```
+
+You can read more in [using multiple providers](../guides/using-multiple-providers) guide.
+
+### Handling the result (`Result` type) {#result-object}
+
+The `perform` method always returns a `Result` type that is either `Ok` or `Err`. This follows the [NeverThrow](https://github.com/supermacro/neverthrow) approach. You can handle the Result in different ways.
+
+#### Conditionals
+
+Use `isOk()` or `isErr()`to check type of result in conditionals.
+
+```typescript
+if (result.isErr()) {
+  // Result is error, error.toString() returns human readable description of what went wrong
+  console.log(result.error.toString());
+} else {
+  // Result is ok and you can access the result data through value
+  console.log(result.value);
+}
+```
+
+#### Matching a value or error
+
+The `Result` provides a `match` method which accepts two functions. The first function handles the `Ok` result and the the second one handles the `Err` result. The conditionals example above can be rewritten using `match` like this:
+
+```typescript
+result.match(
+  value => console.log(value),
+  error => console.log(error.toString())
+);
+```
+
+#### Unsafely unwrapping the result
+
+Lastly, you can just use `unwrap` method which will either return a result value, or throw an error.
+
+```typescript
+try {
+  // Possible error is thrown here and it contains human readable description of what went wrong :)
+  const value = result.unwrap();
+  // You can access value here
+  console.log(value);
+} catch (e) {
+  console.log(e);
+}
+```
+
+## Configuration
+
+The Superface OneSDK is configurable through various environment variables:
```

Comment:
Mention `DEBUG` here as its value essentially configures SDK logging?

Reply:
Good point, I will add it.